cumulusci/core/config/util.py

```diff
@@ -0,0 +1,18 @@
+from cumulusci.core.config import BaseProjectConfig
+from cumulusci.core.exceptions import ServiceNotConfigured
+from cumulusci.core.config.sfdx_org_config import SfdxOrgConfig
+from cumulusci.core.sfdx import get_default_devhub_username
+
+
+def get_devhub_config(project_config: BaseProjectConfig) -> SfdxOrgConfig:
+    """
+    @param project_config: project configuration that has a configured devhub service
```

Comment:
It doesn't need to have a configured devhub service; if it doesn't we get the default devhub from sfdx